test/configuration_unittest.cc

```diff
@@ -58,11 +59,13 @@ TEST_F(MetadataAgentConfigurationTest, PopulatedConfig) {
   ParseConfiguration(
       "ProjectId: TestProjectId\n"
       "MetadataApiNumThreads: 13\n"
-      "MetadataReporterPurgeDeleted: true"
+      "MetadataReporterPurgeDeleted: true\n"
+      "HealthCheckFileDirectory: /a/b/c"
```

Comment:
Let's add `\n` at the end of the string...

Reply:
Done